app/views/api/v1/apps/show.rabl

```diff
@@ -0,0 +1,3 @@
+object @application
+
+attributes :id, :name, :website
```

Comment:
I included ID as a default, but you're right, very little point at moment.

Biggest problem with authorization flow is just lack of documentation, both from Mastodon and Doorkeeper.